config/testdata/.databrickscfg

```diff
@@ -15,14 +15,14 @@ rate_limit=34
 host = https://adb-123.4.azuredatabricks.net/
 
 [azure-pat]
-host = https://adb-123.4.azuredatabricks.net/
+host = https://adb-123.4.azuredatabricks.net
```

Comment:
Shouldn't the canonicalizer take care of this?

Reply:
this is to test canonicalizer